capsules/src/permissions.rs

```diff
@@ -0,0 +1,22 @@
+//! Simple capsule that checks if a permission is set to true or false
+
+#![forbid(unsafe_code)]
```

Comment:
`forbid(unsafe_code)` is already covered in the crate's `lib.rs`, so you don't need to repeat it here.